lib/virt_autotest/utils.pm

```diff
@@ -150,11 +157,13 @@ sub create_guest {
     my $location     = $guest->{location};
     my $autoyast     = $guest->{autoyast};
     my $macaddress   = $guest->{macaddress};
-    my $extra_params = $guest->{extra_params} // "";
+    my $on_reboot    = $guest->{on_reboot}    // "destroy";    # configurable on_reboot policy
```

Comment:
quick question, why used with **"destroy"** as the default for `$on_reboot`. imho, there should be **"reboot"**.

Reply:
This is true, but for some reason the `restart` option doesn't work on old guests.

Reply:
Hmm, but since we anyways fix this later on manually, perhaps we should set it to `restart`. Good catch! Gonna update it now.

Reply:
OK done. I'll restart the VR later on today.

Reply:
yep, I had figured out some product bug on this - **"destroy"** as the default setting for guest `$on_reboot`. 
If the restart option doesn't work on old guests, just separate the old guests use with `destroy`, the other guest use with `reboot` would be made sense here, imho.